### PR TITLE
Run refmt over project & add `yarn fmt` script

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Refmt is Reason's builtin opinionated auto-formatter (like Prettier from JavaScript). 

The main reason I'm starting with this PR is to standardize its formatting with other Reason projects, to make it easier for others to learn from your code. It also will save you and other contributors time from dealing with formatting—just run `yarn fmt` and never worry about indentation again 🎉

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "reason:clean": "bsb -clean-world",
     "webpack:dev": "webpack-dev-server",
     "webpack:build": "NODE_ENV=production webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fmt": "refmt --in-place **/*.re"
```
Comment (Eric-Arellano) [inline frontend/package.json:11]: I can rename this to `yarn format` if you prefer over `yarn fmt`.

**Comment by mbirkegaard** [comment]
Looks good! Merged.

File: frontend/package.json (hunk 1 above)
Comment (mbirkegaard) [reply frontend/package.json]: Looks good! Merged.

**Comment by mbirkegaard** [comment]
Looks good! Merged.

**Pull request merged**